Frame.cc

```diff
@@ -810,18 +812,19 @@ bool Frame::FillSpectralProfileData(int region_id, CARTA::SpectralProfileData& p
                     if (stats_values) {
                         region->FillSpectralProfileData(profile_data, i, *stats_values);
                     } else {
-                        casacore::SubImage<float> sub_image;
+                        std::vector<std::vector<double>> stats_values;
                         std::unique_lock<std::mutex> guard(_image_mutex);
-                        GetRegionSubImage(region_id, sub_image, profile_stokes);
-                        region->FillSpectralProfileData(profile_data, i, sub_image);
```

Comment:
I believe that this `Region` method is now unused, and should be removed. This will leave two very similar methods for filling stats from data which has been passed in (one which accepts a map and one which accepts a vector), and these should probably be merged into one -- but that is not a trivial change and can be done in a separate PR.